metadata.json

```
@@ -14,7 +14,7 @@
     },
     {
       "name": "puppet/extlib",
-      "version_requirement": ">= 3.0.0 < 5.0.0"
+      "version_requirement": ">= 5.0.0 < 6.0.0"
```

Same as https://github.com/theforeman/puppet-pulp/pull/399#discussion_r419539198
```suggestion
      "version_requirement": ">= 3.0.0 < 6.0.0"
```